packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -174,8 +174,8 @@ function runActTests(label, render, unmount, rerender) {
           click();
           click();
         });
-        // it consolidates the 3 updates, then fires the effect
-        expect(Scheduler).toHaveYielded([3]);
```

Comment:
In my opinion, this was undesired behavior. A user clicking three times would see all three updates being flushed not just one. Programmatic click via DOM API would also flush three times: https://codesandbox.io/s/click-is-not-batched-re61e?file=/src/index.js:587-634